dbms/tests/queries/0_stateless/00965_pocopatch_logs_level_bugfix.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
If we have `trace` level in default config, how this test is supposed to test the difference?

Reply:
That's interesting